### DISCUSSION TITLE
怎么关闭转载/复制声明提示? （How to close the repost/copy statement prompt?）

### DISCUSSION THREAD

**Discussion by CRK1918** [mirai-mamori/Sakurairo#657] [Q&A] [answered]
好像每次在highlight.js复制多过两行代码就会显示, 如果我复制一些非常通用的代码就比如说Linux命令的话这个功能是非常烦人的。我能不能把这个转载提示关掉? 

或者至少能不能是一个可选项的开关?

It seems like every time I copy more than two lines of code in highlight.js it shows up, which is very annoying if I copy some very common code like a Linux command. Can I turn off this reprint prompt?

Or at least could it be optional to do so?

**Comment by KotoriK** [comment, marked as answer]
主题设置->页面剪切板版权提示

**Reply by CRK1918** [reply]
谢谢!
